proposals/dao/fip_99.ts

```diff
@@ -0,0 +1,137 @@
+  7 - Whitelist the aave rai pcv deposit on the pcv guardian
+*/
+
+const fipNumber = '99'; // Change me!
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  const priceBoundPSMFactory = await ethers.getContractFactory('PriceBoundPSM');
+  const raiPriceBoundPSM = await priceBoundPSMFactory.deploy(
+    30_000,
+    ethers.constants.WeiPerEther,
+    {
+      coreAddress: addresses.core,
+      oracleAddress: addresses.chainlinkRaiUsdCompositeOracle,
+      backupOracle: ethers.constants.AddressZero,
+      decimalsNormalizer: 0,
+      doInvert: false
+    },
+    0, // mint fee = 0 bp
+    0, // redeem fee = 0 bp
+    ethers.constants.WeiPerEther.mul(10_000_000), // reserves threshold = 10m
+    50_000, // fei rate limit = 50k fei/sec
+    0, // minting buffer cap = 0 = disables minting fei
+    addresses.rai, // rai as the underlying token
+    addresses.feiDAOTimelock // surplus target = timelock
+  );
+  await raiPriceBoundPSM.deployed();
+
+  logging && console.log(`Rai price bound psm deployed`);
+
+  const pcvDripControllerFactory = await ethers.getContractFactory('PCVDripController');
+  const raiPCVDripController = await pcvDripControllerFactory.deploy(
+    addresses.core,
+    addresses.aaveRaiPCVDeposit, // drip source = aave rai pcv deposit
+    raiPriceBoundPSM.address, // drip target = rai psm
+    1800, // drip interval = 30 minutes
+    ethers.constants.WeiPerEther.mul(2_000_000), // drip size = 2 million
+    0 // incentive amount = 0
+  );
+
+  logging && console.log(`Rai pcv drip controller deployed`);
+
+  logging && console.log('Rai PCV drip controller and rai price-bound psm deployed');
+
+  return {
+    raiPriceBoundPSM,
+    raiPCVDripController
+  };
+};
+
+// Do any setup necessary for running the test.
+// This could include setting up Hardhat to impersonate accounts,
+// ensuring contracts have a specific state, etc.
+const setup: SetupUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  console.log(`No actions to complete in setup for fip${fipNumber}`);
+};
+
+// Tears down any changes made in setup() that need to be
+// cleaned up before doing any validation checks.
+const teardown: TeardownUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  console.log(`No actions to complete in teardown for fip${fipNumber}`);
+};
+
+// Run any validations required on the fip using mocha or console logging
+// IE check balances, check state of contracts, etc.
+const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  // Ensure that the global rate limited minter has the MINTER role & the noncustodialpriceboundpsm has the PCV_CONTROLLER role
+  await contracts.core.hasRole(ethers.utils.id('MINTER_ROLE'), addresses.raiPriceBoundPSM);
```

Comment:
Rai PSM should not be able to mint FEI as it is redeem only

Reply:
We might want to enable minting in the future.

Reply:
I'm pretty sure the whole reason for this PSM is to get Rai off the Tribe DAO balance sheet because it is no longer an asset the DAO wants to hold.

Reply:
> it is no longer an asset the DAO wants to hold.

For now! Doesn't make sense to handicap ourselves in the future if we change our mind. Doesn't hurt anything to leave it with the minter role as long as it's paused.

Reply:
Removed.